pkg/restore/util.go

```diff
@@ -36,6 +36,10 @@ func (alloc *idAllocator) Alloc(tableID int64, n uint64, increment, offset int64
 	return alloc.id, alloc.id, nil
 }
 
+func (alloc *idAllocator) AllocSeqCache(sequenceID int64) (min int64, max int64, round int64, err error) {
+	return 0, 0, 0, nil
```

Comment:
Is it safe to return zero if we restore Sequence?

Reply:
do we support backup sequence?

Reply:
i don't think backup sequence need to go through this function